netcore/runtime-llvm.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>runtime.$RID$.Microsoft.NETCore.Runtime.Mono</id>
+        <version>$VERSION$</version>
+        <description>Internal implementation package not meant for direct consumption.  Please do not reference directly. 
+The Mono runtime, and the base library, called mscorlib. It includes the garbage collector, JIT compiler, base .NET data types and many low-level classes.</description>
+        <authors>Microsoft</authors>
+        <projectUrl>https://www.mono-project.com/</projectUrl>
+    </metadata>
+    <files>
+        <file src="System.Private.CoreLib\bin\$COREARCH$\System.Private.CoreLib.dll" target="runtimes\$RID$\native" />
+        <file src="..\mono\mini\.libs\$PLATFORM_AOT_PREFIX$monosgen-2.0$PLATFORM_AOT_SUFFIX$" target="runtimes\$RID$\native" />
+        <file src="..\mono\mini\.libs\$PLATFORM_AOT_PREFIX$monosgen-2.0-llvm$PLATFORM_AOT_SUFFIX$" target="runtimes\$RID$\native" />
```

Comment:
I think producing separate package would be better at this point

Reply:
Yeah we had a meeting where we decided to make a separate package too